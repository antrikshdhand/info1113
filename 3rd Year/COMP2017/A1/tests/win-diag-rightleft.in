place H7
place S19
place B1
place R18
place A2
place Q17
place C3
place P16
place D5
place O15
