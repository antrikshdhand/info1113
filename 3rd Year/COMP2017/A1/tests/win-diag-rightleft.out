White wins!
H7S19B1R18A2Q17C3P16D5O15
Thank you for playing!
